buildpack.md

```diff
@@ -378,6 +381,23 @@ Note that buildpack IDs are expanded depth-first in left-to-right order.
 
 If a buildpack order entry within a group has the parameter `optional = true`, then a copy of the group without the entry MUST be repeated after the original group.
 
+#### Mixin Resolution
```

Comment:
For this PR let's stick to the static requirements expressed in `buildpack.toml` currently and change this PR to only modify the platform API.

Reply:
Addressed in #172, and a new PR will be opened for `buildpack.md` changes